<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPropertiesPaneFeed</class>
 <widget class="QWidget" name="WidgetPropertiesPaneFeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="labelIcon">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCaption">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelDescription">
          <property name="text">
           <string>Description :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelDescriptionValue">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelLink">
          <property name="text">
           <string>Link :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelLinkValue">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelCopyright">
          <property name="text">
           <string>Copyright :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelCopyrightValue">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelLastRefreshed">
          <property name="text">
           <string>Last refreshed :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelLastRefreshedValue">
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelLastError">
          <property name="text">
           <string>Last error :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelLastErrorValue">
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelURL">
          <property name="text">
           <string>Feed URL :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEditURL"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="labelRefresh">
          <property name="text">
           <string>Refresh interval :</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>12</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEditRefreshInterval">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelRefreshIntervalMinutes">
              <property name="text">
               <string>minutes</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="labelRefreshLeaveBlank">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Leave blank to use the global setting</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonSave">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelStatistics">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Statistics</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayoutStats">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelStatAmountOfPosts">
          <property name="text">
           <string>Amount of posts :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelStatAmountOfPostsValue">
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelStatAmountOfFlaggedPosts">
          <property name="text">
           <string>Amount of flagged posts :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelStatAmountOfFlaggedPostsValue">
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelStatOldestPost">
          <property name="text">
           <string>Oldest post :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelStatOldestPostValue">
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelStatNewestPost">
          <property name="text">
           <string>Newest post :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelStatNewestPostValue">
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
